themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -1214,10 +1215,12 @@ group-membership.tooltip=Groups user is a member of. Select a listed group and c
 membership.available-groups.tooltip=Groups a user can join. Select a group and click the join button.
 table-of-realm-users=Table of Realm Users
 view-all-users=View all users
+view-all-clients=View all clients
```

Comment:
The clients page should just display the first list of clients, not requiring clicking view all clients.